features/step_definitions/section_steps.rb

```diff
@@ -151,3 +151,7 @@
 Then('I can see elements from the block') do
   expect(@test_site.home.people).to have_headline_clone
 end
+
+Then('I can see elements from the section composed of shadow roots') do
```

Comment:
composed of a shadow root

Reply:
I fixed, https://github.com/site-prism/site_prism/pull/159/commits/47b8ed05a1cca00dcd1cfb52b9549c14c10cb957